### PR TITLE
Add SwapSpace and XmrExchange.io

### PR DESCRIPTION
**Pull request by Bomper**
I'm a little confused by this file.

1. It's an `.md` file, but all of the markup is HTML?
2. It replicates a lot of the data from https://github.com/monero-project/monero-site/blob/master/_data/merchants.yml. Where is that file published?

### CODE REVIEW

File: community/merchants/index.md [hunk 1]
```diff
@@ -99,10 +99,11 @@ permalink: /community/merchants/index.html
         <h3>Swappers</h3>
         <p>{% t merchants.swappersp %}</p>
         <ul class="logo">
+            <li><a href="https://https://swapspace.co/">SwapSpace.co</a> - instant cryptocurrency exchange aggregator</li>
```
Comment (selsta) [inline community/merchants/index.md:102]:
```suggestion
            <li><a href="https://swapspace.co/">SwapSpace.co</a> - instant cryptocurrency exchange aggregator</li>
```

I don't know if we currently list new exchanges, but in any case, the link is incorrect.

**Comment by erciccione** [comment]
> It replicates a lot of the data from https://github.com/monero-project/monero-site/blob/master/_data/merchants.yml. Where is that file published?

That file needs to be removed, thanks for pointing it out.

@Bomper we do not add centralizes exchanges unless they are known and trusted. The swappers you propose don't fit those requisites. Closing.

**Comment by Bomper** [comment]
> we do not add centralizes exchanges unless they are known and trusted. The swappers you propose don't fit those requisites. Closing.

Where can I read more about the trust that existing exchanges have earned?

I for one would have trouble trusting some of them. Godex for instance uses broken English (About Us sounds downright [ridiculous](https://godex.io/about-us)), makes vacuous claims like "Godex.io is in the list of the best cryptocurrency exchanges." (what list?), can't make up their mind about the number of supported coins (200? 201? 301? at https://godex.io/), has misleading SEO links like "Most Popular Cryptocurrency Exchanges" pointing to a list of [coins](https://godex.io/coins-list/most-popular-cryptocurrency) etc. I'm having serious trouble trusting these guys to code a reliable swap.

ChangeNOW boasts "We respect your privacy / We don’t control your funds / We don’t force you to sign up" on their home page, but have an [entire section on AML/KYC](https://changenow.io/faq/kyc) in their FAQ.

Same for SimpleSwap - "Free from sign-up, limits, complications", then https://simpleswap.io/aml-kyc.

By contrast, SwapSpace truly doesn't require KYC, being an aggregator of swappers. And they're transparent about that - https://swapspace.co/blog/why-swapspace

**Comment by selsta** [comment]
@Bomper In general we add them once they are used in the community. We had issues with people complaining about scammers so we reduced the amount of websites we listed.

Are you the operator of swapspace?

**Comment by Bomper** [comment]
> Are you the operator of swapspace?

No. They just had respectable copy on their website, which can't be said of many swappers.

**Comment by selsta** [comment]
If there is positive feedback from the community I would be in favor of adding it.

**Pull request closed without merging**